bigframes/core/compile/polars/compiler.py

```diff
@@ -268,6 +260,16 @@ def _(self, op: ops.ScalarOp, input: pl.Expr) -> pl.Expr:
             # eg. We want "True" instead of "true" for bool to strin
             return input.cast(_DTYPE_MAPPING[op.to_type], strict=not op.safe)
 
+    # Register ops from other modules
+    from bigframes.operations import IsNullOp, NotNullOp
+    from bigframes.operations.isnull_op import (
+        _polars_isnull_op_impl,
+        _polars_notnull_op_impl,
+    )
+
+    PolarsExpressionCompiler.compile_op.register(IsNullOp, _polars_isnull_op_impl)
+    PolarsExpressionCompiler.compile_op.register(NotNullOp, _polars_notnull_op_impl)
+
```

Comment:
Ugh. I asked Jules not to do this. Instead, we should call polars's `compile_op` from bigframes.operations.isnull_op`